mergify_engine/branch_updater.py

```diff
@@ -159,7 +159,10 @@ def _do_update(ctxt, token, method="merge"):
     except subprocess.CalledProcessError as in_exception:  # pragma: no cover
         for message, out_exception in GIT_MESSAGE_TO_EXCEPTION.items():
             if message in in_exception.output:
-                raise out_exception(in_exception.output.decode())
+                raise out_exception(
+                    "Git reported the following error:\n"
+                    f"```{in_exception.output.decode()}```\n"
```

Comment:
I meant:
```suggestion
                    f"```\n{in_exception.output.decode()}```"
```

Reply:
I think you misread my suggestion again lol